ui/src/components/forms/forms.scss

```diff
@@ -0,0 +1,26 @@
+.contact-form {
```

Comment:
all the properties you have listed here can probably just go on the global css stuff, none of this is super specific to the current form. Also, for padding and margins make sure we are using our space css vars instead of hard coded numbers